view/frontend/templates/checkout/multishipping/billing.phtml

```diff
@@ -0,0 +1,247 @@
+<?php
```

Comment:
This file is overriding a Magento native one, right? Could you add comments to indicate where the modifications are being introduced? Also good idea to add a @see comment pointing to the original one

Reply:
I added @extends comment pointing to the original